docs/stencil-docs/localization/localization-tutorial.md

```diff
@@ -0,0 +1,121 @@
+
+### Prerequisites
+
+In this tutorial, we will use Stencil CLI and [Browsersync](https://github.com/bigcommerce/browser-sync) to serve up a live preview of a theme in development.
+
+To complete this tutorial, you should be familiar with the following concepts:
+
+* [Installing Stencil CLI](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/installing-stencil)
+* [Creating a Stencil CLI Token](https://support.bigcommerce.com/s/article/Store-API-Accounts)
+* [Downloading and uploading custom themes](https://support.bigcommerce.com/s/article/Stencil-Themes#download-upload)
+* [Serving a live preview](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/live-previewing-a-theme#serving-a-live-preview)
+
+
+## Adding a language file
+1. After downloading and extracting your Stencil theme, open the folder containing your theme files and navigate to the `/lang` subfolder. 
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+<!-- theme:  -->
+### Note
+
+The `/lang` subfolder includes the `en.json` file and example language files. The `en.json` file and your language file must be present for a localized Stencil theme to work. Each language requires its own JSON file. 
+
+</div>
+</div>
+</div>
+
+2. Name your translation file based on the [BCP 47 specification](https://tools.ietf.org/html/bcp47) of language and region codes.
+
+3. Copy the desired key-value pairs from the `en.json` file and paste them into your newly-created `es.json` file. Update the values to create your language file.
+
+es.json
+```json
+{
+"header": {
+  "welcome_back": "Bienvenidos, {name}",
+  "skip_to_main": "saltar a la principal"
+   },
+ }
+```
+
+
+## Creating translation keys
+
+Perform the following steps to create new key-value pairs and invoke a defined translation key.
+
+1. Add a key-value pair to a language file.
+  
+  In `en.json`, add `new_hours`.
+
+```json
+"header": {
+        "welcome_back": "Welcome back, {name}",
+        "skip_to_main": "Skip to main content",
+        "new_hours": "Summer hours: {hours}"
+    },
+
+```
+In `es.json`, add `new_hours` and the Spanish translation.
+
+```json
+"header": {
+        "welcome_back": "Bienvenidos, {name}",
+        "skip_to_main": "saltar a la principal",
+        "new_hours": "horario de verano"
+    },
+
+```
+2. Use the handlebars `lang` directive in the appropriate file to show a translated string.
+
+For this example, update `/templates/components/common/header.html` using the code below. Add the code after the `{{/if}}` tag and before the `<header>` tag.
+
+```html
+{{/if}} 
+{{> components/common/alert/alert-success (lang 'header.new_hours' hours="8 AM to 5 PM Central" ) }}
```

Comment:
What does this code look like when executed?
Right now it looks like it would result in the following:

"Summer hours: {hours} hours="8 AM to 5 PM Central"
If this is the case, we don't need the text past the `header.new_hours` part.

Reply:
It reads fine as  (lang 'header.new_hours' hours="8 AM to 5 PM Central" ) will be seen as
summer hours = 8 AM to 5 PM Central

new_hours**:** Summer hours**:** {hours}